roles/marathon/defaults/main.yml

```diff
@@ -1,5 +1,36 @@
 ---
 # defaults file for marathon
 marathon_port: 8080
+hostname: "{{ ansible_all_ipv4_addresses }}"
 consul_dir: /etc/consul.d
+marathon_server_group: marathon_servers
```

Comment:
the terminology is a bit weird here. because down below we're actually building up the url to the zookeeper server group (not the marathon server group). Maybe the variable needs to be - 

marathon_zookeeper_server_group (or marathon_zk_server_group)?